Hi {!GEL_Global_Exception_Log__c.Test_Owner_First_Name__c},

Please note that there is a new exception raised in your process area, please review and update the Global exceptions log as appropriate.

Exception details are as follows.

Status    -   {!GEL_Global_Exception_Log__c.Status__c}
<<<<<<< HEAD
REF Number -  {!GEL_Global_Exception_Log__c.Name}
=======
REF Number -  {!GEL_Global_Exception_Log__c.REF_Number2__c}
>>>>>>> develop

Thank You